ui/imports/shared/popups/AccountsModalHeader.qml

```diff
@@ -1,83 +1,89 @@
 import QtQuick 2.13
-import QtQuick.Controls 2.13
-import QtQuick.Layouts 1.13
-import StatusQ.Controls.Validators 0.1
-
-import utils 1.0
 
 import StatusQ.Controls 0.1
-import StatusQ.Popups 0.1
 import StatusQ.Components 0.1
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Core.Utils 0.1 as StatusQUtils
 
 import "../controls"
-import "../views"
 
-StatusFloatingButtonsSelector {
+StatusComboBox {
     id: root
 
     property var selectedAccount
-    // Expected signature: function(newAccount)
-    property var changeSelectedAccount: function(){}
-    property bool showAllWalletTypes: false
+    property string chainShortNames
+    property int selectedIndex: 0
 
-    repeater.objectName: "accountsListFloatingHeader"
+    QtObject {
+        id: d
+        function getTextColorForWhite(color) {
+            // The grey is kept for backwards compatibility for accounts already created with grey background
+            return color === StatusColors.colors['grey'] || color === StatusColors.colors['white'] ? Theme.palette.black : Theme.palette.white
+        }
+    }
 
-    signal updatedSelectedAccount(var account)
+    control.padding: 0
+    control.spacing: 0
+    control.leftPadding: 8
+    control.rightPadding: 8
+    control.topPadding: 10
 
-    delegate: Rectangle {
-        width: button.width
-        height: button.height
+    control.popup.width: 430
+    control.indicator: null
+
+    control.background: Rectangle {
+        width: contentItem.childrenRect.width + control.leftPadding + control.rightPadding
+        height: 32
```

Comment:
no `width` or `height` needed, the Control takes care of it

Reply:
I was hoping it, but it wouldn't work!! which is why I had to this and the contentItem width too :(

Reply:
Ah of course, it's a different control :) nvm